### PR TITLE
[AIRFLOW-XXX] Specify the default config_file parameter in KubernetesPodOperator do…

### PR DESCRIPTION
**Pull request by leahecole**
…cstring

Make sure you have checked _all_ steps below.
Minor doscstring change in KubernetesPodOperator

### CODE REVIEW

**Review by potiuk — APPROVED** [review]

File: airflow/contrib/operators/kubernetes_pod_operator.py [hunk 1]
```diff
@@ -71,7 +71,8 @@ class KubernetesPodOperator(BaseOperator):
     :type affinity: dict
     :param node_selectors: A dict containing a group of scheduling rules
     :type node_selectors: dict
-    :param config_file: The path to the Kubernetes config file
+    :param config_file: The path to the Kubernetes config file.
+        If not specified, defaults to ~/.kube/config
```
Comment (mik-laj) [inline airflow/contrib/operators/kubernetes_pod_operator.py]:
```suggestion
        If not specified, defaults to ``~/.kube/config``
```
WDYT?

**Comment by leahecole** [comment]
@mik-laj any idea why this build would fail with your small change when it passed before? I'm not familiar with the particular failures in travis but tend to think they're unrelated.

**Comment by mik-laj** [comment]
@leahecole Temporary problems with Travis. Please ignore this.

**Comment by leahecole** [comment]
Alright cool. Up to you lovely folks now to merge if/when you see fit :)

**Comment by mik-laj** [comment]
@leahecole I do not have enough rights yet. We have to wait for other people.

**Comment by leahecole** [comment]
No prob! Perhaps @potiuk can take a look since they already reviewed? (congrats on new committer status!!)

**Review by potiuk — APPROVED** [review]

**Comment by potiuk** [comment]
I think you will need to rebase and push or close/open the PR to trigger rebuild. I cannot merge request with failed tests :(

**Comment by leahecole** [comment]
woop woop, travis is all good.

**Comment by potiuk** [comment]
Can you please squash to one commit as well ? And one more here @kaxil @ashb - I would love to be able to merge ;). Do I need to do something to get the rights to do so ?

**Review by mik-laj — APPROVED** [review]

**Comment by potiuk** [comment]
Now as I have ability to merge, it would be great if you squash the commit and also reword a bit the commit message as the first line is too long @leahecole

**Comment by leahecole** [comment]
Yup - working on it now.

**Comment by leahecole** [comment]
temporarily closing while I work to properly squash things

**Comment by leahecole** [comment]
Moved to #5153 - wasn't allowed to re-open after the rebase and force push. :(

**Pull request closed without merging**